code_samples/logging.js

```diff
@@ -0,0 +1,81 @@
+var winston = require('winston');
+var Config = require('../.').Config;
+var Controller = require('../test/RC');
+var HazelcastClient = require('../.').Client;
+
+
+var winstonAdapter = {
+    logger: new (winston.Logger)({
+        transports: [
+            new (winston.transports.Console)()
+        ]
+    }),
+
+    levels: [
+        'error',
+        'warn',
+        'info',
+        'debug',
+        'silly'
+    ],
+
+    log: function(level, className, message, furtherInfo) {
+        this.logger.log(this.levels[level], className + ' ' + message);
+    }
+};
+
+var loggingHappened = false;
+
+
+winstonAdapter.logger.on('logging', function(transport, level, msg, meta) {     
+    loggingHappened = true;
```

Comment:
You can `console.log` some notification here. Assigning to a variable does not have much visibility from perspective of code sample user.